### PR TITLE
Effect of Control.RightToLeft setting on OpenFileDialog

### PR DESCRIPTION
**Pull request by rpetrusha**
# Effect of Control.RightToLeft setting on OpenFileDialog

## Summary

The documentation doesn't note that, on a right-to-left OS, the containing form's RightToLeft property must be set to RightToLeft.Yes for the dialog to be localized.

## Suggested Reviewers

@Tanya-Solyanik, @mairaw

### CODE REVIEW

**Comment by Tanya-Solyanik** [comment]
@rpetrusha , do you have a workitem that tracks this update?

**Review by Tanya-Solyanik — APPROVED** [review]
Looks good

File: xml/System.Windows.Forms/OpenFileDialog.xml [hunk 1]
```diff
@@ -19,8 +19,10 @@
  This class allows you to check whether a file exists and to open it. The <xref:System.Windows.Forms.OpenFileDialog.ShowReadOnly%2A> property determines whether a read-only check box appears in the dialog box. The <xref:System.Windows.Forms.OpenFileDialog.ReadOnlyChecked%2A> property indicates whether the read-only check box is checked.  
   
  Most of the core functionality for this class is found in the <xref:System.Windows.Forms.FileDialog> class.  
-  
- If you want to give the user the ability to select a folder instead of a file, use <xref:System.Windows.Forms.FolderBrowserDialog> instead.  
+
+On a right-to-left operating system, setting the containing form's <xref:System.Windows.Forms.Control.RightToLeft> property to <xref:System.Windows.Forms.RightToLeft.Yes?displayProperty=nameWithType> localizes the dialog's **File Name**, **Open**, and **Cancel** buttons. If the property is not set to <xref:System.Windows.Forms.RightToLeft.Yes?displayProperty=nameWithType>, English text is used instead.  
+
+ If you want to give the user the ability to select a folder instead of a file,use <xref:System.Windows.Forms.FolderBrowserDialog> instead.
```
Comment (mairaw) [inline xml/System.Windows.Forms/OpenFileDialog.xml]: nit: missing space after comma

**Review by mairaw — APPROVED** [review]
LGTM modulo my comment

**Pull request merged**